README.md

```diff
@@ -1 +1,2 @@
-This repository is for the capstone project of STEP interns Tzirliya Plotkin, Tal Sternberg, and Godsfavour Simon.
\ No newline at end of file
+This repository is for the capstone project of STEP interns Tzirliya Plotkin, Tal Sternberg, and Godsfavour Simon.
+Change to readme
```

Comment:
This is ok for the prototype.  But Ideally we'd want the README to describe the project.  Maybe copy/paste some pieces from the design doc in here.